src/Pages/ProductDetail/RoomInformation/BedType/BedType.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import Title from "../RoomInfoComponent/Title";
+import Svg from "../../SVG/ProductDetailSvg";
+import mixin from "../../../../Styles/mixin";
+
+function BedType({ bedRoom, bedType }) {
+  const [appearCardIdx, setAppearCardIdx] = useState(0);
+  const moveValue = -appearCardIdx * 33.3;
+  return (
+    <BedTypeContainer>
+      <Title marginBottom={"24px"} title={"침대 / 침구유형"} />
+      <Contents>
+        <SliderContainer>
+          <SliderWrapper moveValue={moveValue}>
+            {bedRoom.map((el, idx) => (
+              <ItemWrapper>
+                <Item>
+                  {bed}
+                  <BedRoom>{el}</BedRoom>
+                  <BedInfo>{bedType[idx]}</BedInfo>
+                </Item>
+              </ItemWrapper>
+            ))}
+          </SliderWrapper>
+          {bedRoom.length > 3 && (
+            <>
+              {appearCardIdx !== 0 && (
+                <PrevBtn onClick={() => setAppearCardIdx(appearCardIdx - 1)}>
+                  {prevBtn}
+                </PrevBtn>
+              )}
+              {appearCardIdx < bedRoom.length - 3 && (
+                <NextBtn onClick={() => setAppearCardIdx(appearCardIdx + 1)}>
+                  {nextBtn}
+                </NextBtn>
+              )}
+            </>
+          )}
+        </SliderContainer>
+      </Contents>
+    </BedTypeContainer>
+  );
+}
+
+export default BedType;
+
+const { prevBtn, nextBtn, bed } = Svg;
+
+const BedTypeContainer = styled.article`
+  padding: 48px 0;
+  border-bottom: ${({ theme }) => theme.borderSet};
+`;
+
+const Contents = styled.div`
+  position: relative;
+`;
+
+const SliderContainer = styled.div`
+  overflow: hidden;
+`;
+
+const SliderWrapper = styled.div`
+  ${mixin.flexSet("row", "flex-start")}
+  transform: ${({ moveValue }) => `translateX(${moveValue}%)`};
+  transition: transform 300ms cubic-bezier(0.075, 0.82, 0.165, 1);
+`;
+
+const ItemWrapper = styled.div`
+  flex: 1 0 33.3%;
+  padding: 0 5px;
+`;
+
+const Item = styled.div`
+  padding: 24px;
+  border: ${({ theme }) => theme.borderSet};
+  border-radius: 12px;
+
+  svg {
+    margin-bottom: 16px;
+  }
+`;
+
+const BedRoom = styled.p`
+  margin-bottom: 8px;
+  font-size: 16px;
+  font-weight: ${({ theme }) => theme.fontMedium};
+`;
+
+const BedInfo = styled.p`
+  font-size: 14px;
+  font-weight: ${({ theme }) => theme.fontRegular};
+`;
+
+const Btn = styled.button`
```

Comment:
```suggestion
const Btn = (theme) => css`
```
스타일드 컴포넌트로 만들지 않고 속성 값으로 전달할 수도 있긴 하겠는데 이 경우 큰 차이가 없을거 같습니다.